ansible_collections/arista/avd/roles/eos_designs/templates/network_services/static-routes.j2

```diff
@@ -27,18 +27,6 @@ ipv6_static_routes:
 {%                 endif %}
 {%             endfor %}
 {%         endif %}
-{%         for svi in vrf.svis %}
```

Comment:
For this removal - there is no chance that one of our users was leveraging this before? (even if it looks like we do not support VARP using prefixes).

Maybe @onurgashi can shed some light?

Reply:
It would not work if they did.
Since EOS would not accept the config _if_ we had an address _with_ mask.

Reply:
yeah I see your point. Resolving this as well.